examples/panel-basic/package.json

```diff
@@ -26,6 +26,7 @@
     "@typescript-eslint/parser": "^4.33.0",
     "babel-loader": "^8.2.2",
     "copy-webpack-plugin": "^9.0.1",
+    "cypress-plugin-snapshots": "^1.4.4",
```

Comment:
this one should be removed, one of many tests before getting the "grafana developed" tests to work.